java/mvnd/Portfile

```diff
@@ -31,12 +30,14 @@ long_description mvnd aims at providing faster Maven builds using techniques kno
                 This applies not only to the code coming from Maven plugins and Maven Core,\
                 but also to all code coming from the JDK itself.
 
-github.tarball_from releases
-distname        ${name}-${version}-darwin-amd64
+homepage        https://github.com/apache/maven-mvnd
 
-checksums       rmd160  a20216cb788f0182c4af3249a4a2a8af4ca1a50a \
-                sha256  7a0de6107b9e19290ccc6853ea93a087e5fe1558f64eab66a9e803a03afedc3c \
-                size    26546783
+master_sites    https://downloads.apache.org/maven/mvnd/${version}/
```

Comment:
is there a benefit of moving away from GitHub (I have no opinion here, just curious)?

Reply:
This is the first release since the transfer of the project to the Apache Software Foundation. There is no 0.8.0 release on GitHub.